dev-tools/packer/readme.md.j2

```diff
@@ -0,0 +1,23 @@
+# Welcome to {{.beat_name}} {{.version}}
+
+{{.beat_description}}
+
+## Getting Started
```

Comment:
I'm fine with this because it points users to the doc. I'm not sure how you will deal with versioning, though.